grunt_tasks/build.js

```diff
@@ -103,12 +90,24 @@ module.exports = function (grunt) {
                         'clients/web/src/stylesheets/apidocs/*.styl'
                     ]
                 }
+            }
+        },
+
+        concat: {
+            options: {
+                stripBanners: {
+                    block: true,
+                    line: true
+                }
             },
-            ext: {
+            ext_css: {
                 files: {
                     'clients/web/static/built/ext.min.css': [
-                        'node_modules/eonasdan-bootstrap-datetimepicker/build/css/bootstrap-datetimepicker.css',
-                        'node_modules/jsoneditor/dist/jsoneditor.css'
+                        'node_modules/bootstrap/dist/css/bootstrap.min.css',
+                        'node_modules/bootstrap-switch/dist/css/bootstrap3/bootstrap-switch.min.css',
+                        'node_modules/eonasdan-bootstrap-datetimepicker/build/css/bootstrap-datetimepicker.min.css',
+                        'node_modules/jsoneditor/dist/jsoneditor.min.css',
+                        'node_modules/as-jqplot/dist/jquery.jqplot.min.css'
```

Comment:
I like this change. Every package under the sun ships their own minified css it seems, so this should be fine.